proto/table_marshal.go

```diff
@@ -2944,6 +2946,9 @@ func Marshal(pb Message) ([]byte, error) {
 		// NOTE: This is not efficient.
 		return m.Marshal()
 	}
+	if m, ok := pb.(protov2.Message); ok {
```

Comment:
should use 

var marshalOption = proto.MarshalOptions{
	Deterministic: true,
}
to be safe and consistent